src/core/views.py

```diff
@@ -1288,20 +1296,20 @@ def email_users_proposal(request, proposal_id, user_id):
             )
 
     if (
-            not proposal.owner == user and
-            not proposal.requestor == user and
-            user not in list_of_reviewers
+            not proposal.owner == recipient and
```

Comment:
for readability, this may be a good time to change these to 
```
proposal.owner != recipient and
proposal.requestor != recipient and
...
```
or `recipient not in (proposal.owner, proposal.requestor) and ...`

Reply:
Yeah, this was low-hanging fruit (which is quite abundant in Rua's codebase).

I went with:
```
if recipient not in {proposal.owner, proposal.requestor} | reviewers:
```

Reply:
Nice use of sets. Much cleaner :+1:

Reply:
Sadly it's now `if recipient in {proposal.owner, proposal.requestor} | set(reviewers):`
Still an improvement.

Reply:
should it be `in` or `not in`? :thinking: